blueprints/ember-cli-flash/index.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   afterInstall: function() {
     var TEST_HELPER_PATH = 'tests/test-helper.js';
     var IMPORT_STATEMENT = EOL + "import './helpers/flash-message';";
-    var INSERT_AFTER = "import resolver from './helpers/resolver';";
+    var INSERT_AFTER = "import { start } from 'ember-qunit';";
```

Comment:
Does this break on versions older than 2.17? Can we make this a ternary based on the version?

Reply:
It does only work on >=2.17:

> This should support any version >= 2.17, which hopefully is good enough.

FWIW, this addon has never installed correctly on those newer versions. But if there is an easy way to do an Ember version check within a blueprint, I agree it's worth doing.

Reply:
We should be able to do it using `ember-cli-version-checker`:
https://github.com/ember-cli/ember-cli-version-checker